docs/app/pages/components/components-sections/tables/column-picker/snippets/sort.ts

```diff
@@ -0,0 +1,5 @@
+sort = (a: ColumnPickerGroupItem, b: ColumnPickerGroupItem) => {
+    const aCombined = a.group ? `${a.group}${a.name}` : a.name;
+    const bCombined = b.group ? `${b.group}${b.name}` : b.name;
```

Comment:
I think I might still need it. If `a.group` is undefined then the result would be `undefined${a.name}`, which would result in incorrect sorting.

Reply:
Sorry, I misread the updated code, you are right.

I've noticed that it has a failure case if you also have a property with the same name as a group. I think this is a problem with the code that converts the columns into tree nodes, however. If we allow arbitrary sorting, we will have to account for the fact that all items with the same group might not end up being consecutive, so the code that deals with the output may have to be more tolerant.

In pseudocode, it could be something like
```
for each column
  if column is group item
    groupNode = createOrFindGroupNode(column, treeData)
    groupNode.children.push(createTreeNode(column))
  else
    treeData.push(createTreNode(column))
```